exercicios/para-casa/entregas/exemplo-nome-sobrenome/Personagem.js

```diff
@@ -0,0 +1,44 @@
+
+
+
+import { SuperPoder } from './SuperPoder.js';
+
+export class Personagem {
+  #nome;
+  #nomeVidaReal;
+  #poderes = [];
+
+  constructor(nome, nomeVidaReal) {
+    this.#nome = nome;
+    this.#nomeVidaReal = nomeVidaReal;
+  }
+
+  get nome() {
+    return this.#nome;
+  }
+
+  get nomeVidaReal() {
+    return this.#nomeVidaReal;
+  }
+
+  adicionaSuperPoder(superpoder) {
+    if (!(superpoder instanceof SuperPoder)) {
+      console.log("Sem autorização");
+      return;
+    }
+
+    if (this.#poderes.length <= 4) {
```

Comment:
dá pra colocar mais um if pra se assegurar sobre o numero maximo de poderes(4) nesse, nomeando esse numero maximo como uma constante, por exemplo: 

 if (this.#poderes.length < MAX_PODERES) {
        //bota o codigo aqui
    } else {
        console.log("Personagem só pode ter no máximo 4 poderes");
    }

Reply:
Vou fazer essa alteração, não me atentei a esse detalhe. Obrigada :)